{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -76,8 +76,10 @@
 
 # Celery
 # ------------------------------------------------------------------------------
+{% if cookiecutter.use_docker == 'n' - %}
```

Comment:
The build failed as there is a syntax error here. Should not be any spaces between the `-` and the closing `%}`:

```suggestion
{% if cookiecutter.use_docker == 'n' -%}
```